nixos/doc/manual/installation/installing.xml

```diff
@@ -326,10 +326,11 @@ Retype new UNIX password: ***
     </screen>
     <note>
      <para>
-      To prevent the password prompt, set
+      During unattanded installations, ie. with the option
```

Comment:
`s/unattanded/unattended/`

* * *

"ie." would mean "During unattended installations, *in other words* with the option [mutableUsers = false][...]".

This may be fine, though "unattended installation" does not (forcibly) imply "mutableUsers = false".

What do you think of:

> During unattended installations, where no root password is expected to be given interactively, it is possible to use [...]

This, though, loses the `mutableUsers = false` hint, but makes the statement more global; It is just as well possible to want a `mutableUser = true` installation be installed unattended, right?

Feel free to consider other alternatives, including and excluding the configuration hints.